rero_ils/modules/patron_transactions/utils.py

```diff
@@ -126,17 +126,16 @@ def create_patron_transaction_from_overdue_loan(
             'note': _('incremental overdue fees'),
             'total_amount': total_amount,
             'creation_date': datetime.now(timezone.utc).isoformat(),
+            'steps': [
+                {'timestamp': fee[1].isoformat(), 'amount': fee[0]}
```

Comment:
Not nice to use a set for fee. Better to use a dictionary like:
```python
{
    'amount': fee_amount,
    'day': day
}
```
then we can use `fee['day']` instead of `fee[1]`

Reply:
To be corrected later, as it affects several parts